extensions/amp-bind/0.1/test/test-bind-expression.js

```diff
@@ -205,6 +205,17 @@ describe('BindExpression', () => {
       expect(evaluate('+"1"')).to.equal(1);
     });
 
+    it('should parse special characters', () => {
+      expect(evaluate('"\\n"')).to.equal('\n');
```

Comment:
What does this look like when written in HTML?

Reply:
Looks like `"\n"`.